pkg/controllermanager/deployer/deployer.go

```diff
@@ -790,17 +810,14 @@ func (deployer *Deployer) populateDescriptions(base *appsapi.Base) error {
 
 	descTemplate := &appsapi.Description{
 		ObjectMeta: metav1.ObjectMeta{
+			Name:      base.Name,
```

Comment:
So we're using a single name for both helm and non helm descriptions? We're allowing defining helm charts and other resource together in a subscription.